dnf/cli/commands/repolist.py

```diff
@@ -139,10 +139,10 @@ def run(self):
                 enabled = True
                 if arg == 'disabled':
                     continue
-                if any((include_status, verbose, 'repoinfo' in self.opts.command)):
-                    ui_enabled = ehibeg + _('enabled') + hiend
-                    ui_endis_wid = exact_width(_('enabled'))
-                if verbose or ('repoinfo' in self.opts.command):
+                if verbose or self.opts.command == 'repoinfo':
+                    if include_status:
+                        ui_enabled = ehibeg + _('enabled') + hiend
+                        ui_endis_wid = exact_width(_('enabled'))
```

Comment:
Here the behaviour of `dnf repolist --all`  command (i.e. verbose=False, include_status=True, command='repolist') has changed - lines
```
ui_enabled = ehibeg + _('enabled') + hiend
ui_endis_wid = exact_width(_('enabled'))
```
are not executed any more.

Reply:
Ooops! Sorry about this one. Logic is hard! 🤦

Just went back to the original separate `if` statements. Still replaced the `any(...)` with `or`s since I believe that still makes sense.

Reply:
Thanks, I also consider `or` statement clearer and more readable.